test/config/utility.cc

```diff
@@ -145,6 +145,14 @@ std::string ConfigHelper::startTlsConfig() {
                   TestEnvironment::runfilesPath("test/config/integration/certs/serverkey.pem")));
 }
 
+std::string ConfigHelper::testInspectorFilter() {
+  return R"EOF(
+name: "envoy.filters.listener.test"
+typed_config:
+  "@type": type.googleapis.com/google.protobuf.Struct
```

Comment:
Probably use the typed config here, since using name to lookup extension is deprecated soon https://github.com/envoyproxy/envoy/pull/20397

Reply:
oops, sorry, just saw you guys already discussed this at https://github.com/envoyproxy/envoy/pull/20571/files#r837772893